examples/cpp/CMakeLists.txt

```diff
@@ -14,5 +14,5 @@
 # limitations under the License.
 # ******************************************************************************
 message(STATUS "C++ Examples will be built")
-add_subdirectory(infer_multiple_networks)
-add_subdirectory(infer_single)
+#add_subdirectory(infer_multiple_networks)
+#add_subdirectory(infer_single)
```

Comment:
Commented code. Should this be removed?

Reply:
Done. Reorganized the cpp directory to have a new `inference` directory with the contents of test/benchmark moved.